File: monitor/events/list-get-example-sourceipaddress-filter/list-get-example-sourceipaddress-filter.4.x.cs
// Download the twilio-csharp library from twilio.com/docs/csharp/install
using System;
using Twilio.Monitor;
class Example
{
  static void Main(string[] args)
  {
    // Find your Account Sid and Auth Token at twilio.com/user/account
    string AccountSid = "ACXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX";
    string AuthToken = "your_auth_token";
    var monitor = new MonitorClient(AccountSid, AuthToken);

    var options = new EventListRequest();
    options.SourceIpAddress = "104.14.155.29";
    options.StartDate = new DateTime(2015, 4, 25);
    options.EndDate = new DateTime(2015, 4, 25, 23, 59, 59);
    var events = monitor.ListEvents(options);
    foreach (var e in events.Events)
    {
      Console.WriteLine(e.Description);
    }
  }
}

File: monitor/events/list-get-example-sourceipaddress-filter/list-get-example-sourceipaddress-filter.4.x.rb
# Get twilio-ruby from twilio.com/docs/ruby/install
require 'twilio-ruby'

# Get your Account Sid and Auth Token from twilio.com/user/account
account_sid = 'ACXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX'
auth_token = 'your_auth_token'
@client = Twilio::REST::MonitorClient.new account_sid, auth_token

@params = {
  source_ip_address: '104.14.155.29',
  start_date: '2015-04-25T00:00:00Z',
  end_date: '2015-04-25T23:59:59Z'
}
@client.events.list(@params).each do |e|
  puts e.description
end
